packages/node_modules/@webex/internal-plugin-device/test/unit/spec/features/feature-model.js

```diff
@@ -56,6 +56,117 @@ describe('plugin-devices', () => {
       });
     });
 
+    describe('#parse()', () => {
+      let fixture;
+      let rto;
+
+      beforeEach('initialize the feature model', () => {
+        featureModel = new FeatureModel();
+      });
+
+      describe('when the pto is not defined', () => {
+        beforeEach('get the rto', () => {
+          rto = featureModel.parse(fixture);
+        });
+
+        it('should return an empty object', () =>
+          assert.deepEqual(rto, {}));
+      });
+
+      describe('when the pto is defined', () => {
+        beforeEach('define the fixture', () => {
+          fixture = {};
+        });
+
+        describe('when the value is a number', () => {
+          let num;
+
+          beforeEach('set the value to a number', () => {
+            fixture.val = '1234';
+            num = Number(fixture.val);
+            rto = featureModel.parse(fixture);
+          });
+
+          it('should set the value to a instance of number', () =>
```

Comment:
In this tests we have one assertion per test, but on others there are multiple assertions. I would like to keep the testing consistent. which pattern are we following?

Reply:
I believe I set this up this way to test more aggressively at the lowest child class object, as all upstream classes depend on the consistency of this class. I can refactor this if needed.

Reply:
Personally, I prefer one assertion per test, but I saw some other code in this PR that does not follow that format